assets/blocks/quiz/quiz-block/use-has-questions.js

```diff
@@ -16,28 +16,39 @@ export const useHasQuestions = ( clientId ) => {
 			.filter( ( block ) => !! block.attributes.title )
 	);
 
+	const { editPost } = useDispatch( 'core/editor' );
+	const setQuizHasQuestions = useCallback(
+		( on ) => {
+			const { _quiz_has_questions: savedValue } =
```

Comment:
And here maybe `hasQuestionsMeta`, instead of `savedValue`?

Reply:
Tried to clear up the naming in a few places, and renamed the whole thing to `useUpdateQuizHasQuestionsMeta` to put everything in context. This is now `currentValue`, following Gutenberg's naming. (This is the meta value that's saved for the post, without the pending edits)